src/common.rs

```diff
@@ -318,6 +318,45 @@ impl ConnectionConfig {
     }
 }
 
+pub fn check_account_id(
+    connection_config: ConnectionConfig,
+    account_id: String,
+) -> color_eyre::eyre::Result<Option<near_primitives::views::AccountView>> {
+    let query_view_method_response = actix::System::new().block_on(async {
+        near_jsonrpc_client::new_client(connection_config.rpc_url().as_str())
+            .query(near_jsonrpc_primitives::types::query::RpcQueryRequest {
+                block_reference: near_primitives::types::Finality::Final.into(),
+                request: near_primitives::views::QueryRequest::ViewAccount { account_id },
+            })
+            .await
+    });
+    match query_view_method_response {
+        Ok(rpc_query_response) => {
+            let account_view =
+                if let near_jsonrpc_primitives::types::query::QueryResponseKind::ViewAccount(
+                    result,
+                ) = rpc_query_response.kind
+                {
+                    result
+                } else {
+                    return Err(color_eyre::Report::msg(format!("Error call result")));
+                };
+            Ok(Some(account_view))
+        }
+        Err(_) => return Ok(None),
+    }
+}
```

Comment:
Let's implement a retry logic, so we don't break out of an interactive mode immediately after the first error that could happen due to some network glitch. Retry 5 times with 1 second delay before returning non-Ok(Some) result.

Also, we should notify users that CLI is fetching the data from the server, and so a user should expect to wait a little bit.